gcp-cloud-run/README.md

```diff
@@ -0,0 +1,120 @@
+# Piiano vault deployment on GCP
```

Comment:
@david-piiano This page is not in the same structure as the other modules. Can you please suggest how you would like this page to change?

Reply:
I am aligning it to what we did for AppRunner. See changes to readme